### PR TITLE
Save transformer model to ONNX

### PR DESCRIPTION
**Pull request by ClaireDons**
This pull request adds the following:
- [X] Add functions so that workflow is easier to follow
- [X] Save model to ONNX
- [X] Load saved model with test set and check results
- [x] Add more explanation to the workflow

### CODE REVIEW

**Review by BSchilperoort — APPROVED** [review]
Nice! The train/test loss plot looks a lot better! Much less overfitting. Still not a lot of predictive power but that's to be expected.
 
Maybe add to the introduction that this notebook does not make use of dimensionality reduction, unlike the others? That's a pretty significant difference in the model layout.
